bin/create-k8s-config-maps-and-secrets

```diff
@@ -16,7 +18,7 @@ metadata:
 data:
   AWS_DEFAULT_REGION: "${AWS_DEFAULT_REGION}"
   USE_LOCALSTACK: "${USE_LOCALSTACK}"
-  LOCALSTACK_HOST: "${BACKEND_HOST}"
```

Comment:
Is BACKEND_HOST still used (and needed in env-defaults)?

Reply:
yes